appengine/geddy/package.json

```diff
@@ -8,8 +8,9 @@
     "node": "~0.12.7"
   },
   "scripts": {
-    "start": "node node_modules/geddy/bin/cli.js",
-    "debug": "node node_modules/geddy/bin/cli.js --debug",
+    "prestart": "GEDDY_SECRET=config/secrets.json; [[ -f $GEDDY_SECRET ]] || echo '{}' > $GEDDY_SECRET && node node_modules/.bin/geddy gen secret",
```

Comment:
:/ I'm not a fan of having our testing infrastructure leak into our samples like this.